data/json/scenarios/scenarios.json

```diff
@@ -168,5 +168,14 @@
     "allowed_locs" : ["field"],
     "map_special": "mx_helicopter",
     "flags": ["HELI_CRASH"]
-    }
+  },{
+    "type": "scenario",
+    "ident": "Mine_bottom",
+    "name": "Bottom Of A Mine",
+    "points" : 1,
+    "description": "You were mining along when you found...something. You're not sure what, but it sure is dark down here.",
```

Comment:
Hmm. Perhaps as separate profession? It would certainly fit, but i feel something like that probably shouldn't be restricted to a mine, it might be best saved for a different scenario, when there are more interesting things to explore in caves perhaps?
I certainly wouldn't be upset if someone else wanted to tweak the miner to be more a spelunker though.